src/Sylius/Bundle/AdminBundle/Tests/Command/ChangeAdminUserPasswordCommandTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Tests\Command;
+
+use PHPUnit\Framework\TestCase;
+use Sylius\Bundle\AdminBundle\Command\ChangeAdminUserPasswordCommand;
+use Sylius\Component\Core\Model\AdminUser;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+use Sylius\Component\User\Security\PasswordUpdaterInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Tester\CommandTester;
+
+final class ChangeAdminUserPasswordCommandTest extends TestCase
```

Comment:
We're missing a test here for failing the non interactive command

Reply:
True. It's also missing for the test for `CreateAdminUserCommand `. Can add that one as well in a separate PR.

Reply:
I see there're still no test for failing in case of non-interactive way. Can you add it for a class you've created (and, if you want, create tests for the `CreateAdminUserCommand` in a separate PR)?